ui/app/templates/components/shamir-modal-flow.hbs

```diff
@@ -1,194 +1,197 @@
-<Modal
-  @title="Generate operation token"
-  @onClose={{action 'onClose'}}
-  @isActive={{isActive}}
-  @type="warning"
-  @showCloseButton={{true}}
->
-  <section class="modal-card-body">
-    {{#if encoded_token}}
-      <p class="has-bottom-margin-l">
-        {{#if otp}}
-        There are three values below. The encoded operation token and the OTP are used in the <span class="has-text-semibold">DR operation token command</span>, which you will need to copy and paste into your CLI in order to generate the operation token. Save your encoded operation token and OTP.
-        {{else}}
-        The encoded operation token and the OTP are used in the <span class="has-text-semibold">DR operation token command</span>, which you will need to copy and paste into your CLI in order to generate the operation token. Save your encoded operation token and OTP.
-        {{/if}}
-      </p>
-      <div class="has-bottom-margin-m">
-        <h4 class="title is-7 has-bottom-padding-m is-fullwidth">
-          Encoded operation token
-        </h4>
-        <div class="message is-list has-copy-button" tabindex="-1">
-          <HoverCopyButton @copyValue={{encoded_token}} />
-          <code class="is-word-break" data-test-shamir-encoded-token>{{encoded_token}}</code>
-        </div>
-        {{#if otp}}
-          <h4 class="title is-7 has-bottom-padding-m is-fullwidth">
-            One time password (OTP)
-          </h4>
-          <div class="message is-list has-copy-button" tabindex="-1">
-            <HoverCopyButton @copyValue={{otp}} />
-            <code class="is-word-break">{{otp}}</code>
-          </div>
-        {{/if}}
+{{#if isActive}}
+  <Modal
+    @title="Generate operation token"
+    @onClose={{action 'onClose'}}
+    @isActive={{isActive}}
+    @type="warning"
+    @showCloseButton={{true}}
+    @modalId="shamir"
+  >
+    <section class="modal-card-body">
+      {{#if encoded_token}}
+        <p class="has-bottom-margin-l">
+          {{#if otp}}
+          There are three values below. The encoded operation token and the OTP are used in the <span class="has-text-semibold">DR operation token command</span>, which you will need to copy and paste into your CLI in order to generate the operation token. Save your encoded operation token and OTP.
+          {{else}}
+          The encoded operation token and the OTP are used in the <span class="has-text-semibold">DR operation token command</span>, which you will need to copy and paste into your CLI in order to generate the operation token. Save your encoded operation token and OTP.
+          {{/if}}
+        </p>
+        <div class="has-bottom-margin-m">
           <h4 class="title is-7 has-bottom-padding-m is-fullwidth">
-            DR operation token command
+            Encoded operation token
```

Comment:
I honestly don't know what that's showing as different.  I double-checked and I only added a conditional.  I'll investigate further if concept is approved.